LinkedIdeas/CanvasViewController.swift

```diff
@@ -219,15 +305,31 @@ extension CanvasViewController {
   }
   
   override func mouseDragged(with event: NSEvent) {
+    if (dragCount <= 3) {
+      Swift.print("[mouseDragged]")
```

Comment:
Nifty, I didn't know you could do that. I always relied on `NSLog` when doing `macOS`.
TIL